api/src/pcapi/use_cases/save_venue_bank_informations.py

```diff
@@ -104,11 +100,17 @@ def get_referent_venue(self, application_details: ApplicationDetail, offerer: Of
 
         if siret:
             venue = self.venue_repository.find_by_siret(siret)
-            check_venue_presence(venue)
+            if not venue:
+                raise CannotRegisterBankInformation("Venue not found")
+            if not api_entreprises.check_siret_is_still_active(siret):
+                raise CannotRegisterBankInformation("SIRET is not active any longer")
```

Comment:
(détail)

```suggestion
                raise CannotRegisterBankInformation("SIRET is no longer active")
```

... mais c'est vraiment parce que je ne trouve rien de passionnant à redire sur cette pull request. :)